uber/templates/registration/check_in_form.html

```diff
@@ -0,0 +1,73 @@
+<table class="check-in">
+    <tr>
+        <td>Name:</td>
+        <td>{{ attendee|form_link }}</td>
+    </tr>
+    {% if c.COLLECT_EXACT_BIRTHDATE %}
+        <tr>
+            <td>Birthdate</td>
+            <td>
+                <input type="text" id="checkin-birthdate" class="date" value="{{ attendee.birthdate|date:"Y-m-d"}}" />
+            </td>
+        </tr>
+         {% if attendee.birthdate %}
+            <tr>
+                <td>Age Group:</td>
+                <td>{{ attendee.age_group_conf.desc }}</td>
+            </tr>
+        {% endif %}
+    {% else %}
+        <tr>
+            <td>Age Group:</td>
+            <td>
+                <select id="checkin-age">
+                    {% options c.AGE_GROUP_OPTS attendee.age_group %}
+                </select>
+            </td>
+        </tr>
+    {% endif %}
+    {% if attendee.paid == c.PAID_BY_GROUP and not attendee.group_id %}
+        <tr>
+            <td>Group:</td>
+            <td>
+                <select id="checkin-group">
+                    <option value="">No Group</option>
+                    {% options groups attendee.group_id %}
+                </select>
+            </td>
+        </tr>
+    {% endif %}
+    <tr>
+        <td>Email:</td>
+        <td>{{ attendee.email }}</td>
+    </tr>
+    <tr>
+        <td>Zipcode:</td>
+        <td>{{ attendee.zip_code }}</td>
+    </tr>
+    <tr>
+        <td>Emergency Contact:</td>
+        <td>{{ attendee.ec_phone }}</td>
+    </tr>
+    <tr>
+        <td>Badge Type:</td>
+        <td>{{ attendee.badge_type_label }}
+            {% if attendee.ribbon != c.NO_RIBBON %}
+                ({{ attendee.ribbon_label }})
+            {% endif %}
+        </td>
+    </tr>
```

Comment:
I didn't add in the "This person has a badge holder" row yet, because that will need to go into the MAGPrime repo anyway.